packages/gatsby-plugin-treat/package.json

```diff
@@ -16,5 +16,10 @@
   "dependencies": {
     "babel-plugin-treat": "^1.6.2"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "gatsby": "^2.18.11",
+    "treat": "*",
+    "wait-for-localhost": "^3.3.0"
```

Comment:
Looks like you removed the use of this library in the test, still needed elsewhere?